app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    cached_calcs = {}
+
+    def wrapper(*args, **kwargs) -> int | float:
+        if args not in cached_calcs:
+            print("Calculating new result")
```

Comment:
Remove the print

Reply:
The task tells that prints should be.  Tests fails whiout them